float64 Pitch
float64 Yaw
float64 X
float64 PitchRate
float64 YawRate
float64 V
float64 Time
